dh/sidh/internal/p434/core.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // -----------------------------------------------------------------------------
-// Functions for traversing isogeny trees acoording to strategy. Key type 'A' is
+// Functions for traversing isogeny trees according to strategy. Key type 'A' is
```

Comment:
it's possible that these are from third party libraries. I'm happy to try to send PRs to them, although I have a rather long queue and make no promises about when I'll reach them.

In any case, if the files don't originate here, if you can give me a pattern describing files I shouldn't touch, I'm happy to drop them.

Reply:
this code is generated by file /circl/dh/sidh/templates/core.gotemp